helm/charts/hpe-csi-driver/README.md

```diff
@@ -21,34 +20,20 @@ The following table lists the configurable parameters of the HPE-CSI chart and t
 |---------------------------|-------------------------------------------------------------|-------------|
-| backendType                   | Name of the HPE storage backend type (nimble, primera3par).                 | nimble |
-| secret.create                   | Enable creation of a `Secret` along with CSP deployment.                 | true |
-| secret.backend                   | HPE storage backend hostname or IP address.                 | 192.168.1.1 |
-| secret.username                  | Username for the backend.                                   | admin       |
-| secret.password                  | Password for the backend.                                   | admin       |
-| crd.nodeInfo.create       | Create `hpenodeinfo` CRDs required by HPE CSI Driver. Should only be enabled with Helm 2, as they are automatically created with Helm 3 without this flag.                  | false        |
-| crd.volumeInfo.create       | Create `hpevolumeinfo` CRDs required by HPE CSI Driver for 3PAR and Primera. Should only be enabled with Helm 2, as they are automatically created with Helm 3 without this flag. | false        |
 | logLevel             | Log level. Can be one of `info`, `debug`, `trace`, `warn` and `error`.                                        | info         |
 | imagePullPolicy | Image pull policy (`Always`, `IfNotPresent`, `Never`).                                          | IfNotPresent |
 | disableNodeConformance | Disable automatic installation of iSCSI/Multipath Packages.                                           | false |
-| storageClass.name  | The name to assign the created `StorageClass`.                                          | hpe-standard |
-| storageClass.create | Enables creation of a `StorageClass` managed by this Helm chart.                            | true        |
-| storageClass.defaultClass | Whether to set the created `StorageClass` as the clusters default `StorageClass`.                                | false       |
-| storageClass.parameters.fsType                    | Type of file system being used (ext4, ext3, xfs, btrfs).     | xfs         |
-| storageClass.parameters.volumeDescription         | Default volume description set on backend volume.     | -         |
-| storageClass.parameters.accessProtocol            | Access protocol to use for storage connectivity (iscsi, fc).     | iscsi         |
+| iscsi.chapUser | Username for iSCSI CHAP authentication.                                           | "" |
+| iscsi.chapPassword | Password for iSCSI CHAP authentication. Packages.                             | "" |
 
 It's recommended to create a [values.yaml](https://github.com/hpe-storage/co-deployments/blob/master/helm/values/csi-driver) file from the corresponding release of the chart and edit it to fit the environment the chart is being deployed to. Download and edit [a sample file](https://github.com/hpe-storage/co-deployments/blob/master/helm/values/csi-driver).
```

Comment:
Can we remove this? It's not really necessary given we've cut the parameters down.

Reply:
the problem is if they use old values.yaml with new releases. Since they are incompatible this was added.